src/Features/CSharp/Portable/Debugging/CSharpProximityExpressionsService.cs

```diff
@@ -100,6 +100,8 @@ public CSharpProximityExpressionsService()
             }
         }
 
+        public static IList<string> GetProximityExpressions(SyntaxTree syntaxTree, int position) => Do(syntaxTree, position, cancellationToken: default);
```

Comment:
If we do keep this (might be removed from https://github.com/dotnet/roslyn/pull/50718/files#r569757295), it needs to have a proper `CancellationToken` parameter (caller may pass `CancellationToken.None`, but it's not going to do so by default).

Reply:
Can add.

Out of curiosity, is there history behind passing in the cancellation tokens to these synchronous methods? Aka do we really expect the code to have a high perf cost? I imagine that the underlying bits aren't doing Task.Result which would then infer that they're doing "throwIfCancellationRequested" in tight loops.